recipe/meta.yaml

```diff
@@ -1,6 +1,6 @@
 {% set version = "1.6.3" %}
-{% set number = 0 %}
-{% set faiss_proc_type = "gpu" if cuda_compiler_version != "None" else "cpu" %}
+{% set number = 1 %}
+{% set faiss_proc_type = "cuda" if cuda_compiler_version != "None" else "cpu" %}
```

Comment:
Arguably, the build number wouldn't need to be increased, as it doesn't touch anything about non-gpu builds.

The change in extension is following the discussion in https://github.com/conda-forge/conda-forge.github.io/issues/1059.